openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/base/components/UserMenu.vue

```diff
@@ -92,6 +98,27 @@ export default {
     login: function () {
       OpenC3Auth.login(location.href)
     },
+    role: function () {
+      if (this.username === 'Anonymous') {
+        return 'Admin'
+      } else {
+        return [
+          ...new Set( // Use Set to remove duplicates
+            OpenC3Auth.userroles()
+              // Roles are like ALLSCOPES__custom DEFAULT__viewer
+              // but it also includes default-roles-openc3
+              .map((element) => element.split('__')[1])
+              .filter(Boolean), // Get rid of non roles (default-roles-openc3)
+          ),
+        ]
+          .map((element) => self.capitalize(element))
+          .sort()
+          .join(', ')
```

Comment:
I'm just showing all the roles. Astro icon will just truncate the list if it gets too large but our default roles work really well like "Admin, Viewer" or "Operator, Viewer".